crates/ruff_linter/src/rules/pylint/rules/bad_str_strip_call.rs

```diff
@@ -72,10 +74,20 @@ pub(crate) enum ValueKind {
 }
 
 impl ValueKind {
-    fn from(expr: &Expr) -> Option<Self> {
+    fn from(expr: &Expr, semantic: &SemanticModel) -> Option<Self> {
         match expr {
             Expr::StringLiteral(_) => Some(Self::String),
             Expr::BytesLiteral(_) => Some(Self::Bytes),
+            Expr::Name(name) => {
+                let binding_id = semantic.only_binding(name)?;
+                let binding = semantic.binding(binding_id);
+
+                match () {
+                    () if typing::is_string(binding, semantic) => Some(Self::String),
+                    () if typing::is_bytes(binding, semantic) => Some(Self::Bytes),
+                    () => None,
+                }
```

Comment:
Isn't that the same as a `if typign::is_string() {...} else if typing::is_bytes {}`?